integr8ly/installation
# Reset upgrade playbook to only include prereq version check, manifest…

… update & monitoring upgrade

## Additional Information
Follow up on #1037 

## Verification Steps
1. Install from release-1.5.0 tag
2. Verify all alerts & green
3. Upgrade using this branch
4. Verify new alerts from #1037 are installed
